tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -2,8 +2,8 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
-@pytest.mark.integration_test
+#@pytest.mark.skip
+#@pytest.mark.integration_test
```

Comment:
For integration tests, we want to leave the decorator `@pytest.mark.integration_test` uncommented. This decorator tells pytest that this is an integration test, so we should run it after all of the unit tests.